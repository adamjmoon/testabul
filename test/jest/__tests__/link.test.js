import React from 'react';
import { Router, Route, Link } from 'react-router'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Link page="http://www.facebook.com">Facebook</Link>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
