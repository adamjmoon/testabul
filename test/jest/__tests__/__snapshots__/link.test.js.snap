exports[`test renders correctly 1`] = `
<a
  onClick={[Function bound handleClick]}
  page="http://www.facebook.com"
  style={Object {}}>
  Facebook
</a>
`;
